### PR TITLE
Combine linting fe jobs into one

### PR DESCRIPTION
**Pull request by uladzimirdev**
## Descripton

even if we do not run out of runners, it's not very optimal to have heavy repeatable steps for 3 very light jobs, so I combined them into one (spin off of the job is more heavy than running ts-check itself).

### Note for reviewers
after merge we'll need to update checks at github settings - required checks are not updated automatically

### CODE REVIEW

**Comment by nemanjaglumac** [comment]
Merging prettier and eslint doesn't seem like a bad idea, but I'd still leave type-check as a separate job.

File: package.json [hunk 1]
```diff
@@ -291,11 +291,14 @@
     "concurrently": "yarn && concurrently --kill-others -p name",
     "dev": "yarn && yarn clean:cljs && concurrently -n 'backend,frontend,cljs,static-viz' -c 'blue,green,yellow,magenta,red' 'clojure -M:run' 'yarn build-hot:js-wait' 'yarn build-hot:cljs' 'yarn build-static-viz:watch-wait'",
     "dev-ee": "yarn && yarn clean:cljs && concurrently -n 'backend,frontend,cljs,static-viz' -c 'blue,green,yellow,magenta,red' 'clojure -M:run:ee' 'MB_EDITION=ee yarn build-hot:js-wait' 'MB_EDITION=ee yarn build-hot:cljs' 'yarn build-static-viz:watch-wait'",
-    "type-check": "yarn clean:cljs && yarn build:cljs && tsc --noEmit",
+    "type-check": "yarn clean:cljs && yarn build:cljs && yarn type-check-pure",
+    "type-check-pure": "tsc --noEmit",
```
Comment (iethree) [inline package.json:295]: ❤️  I set up an alias to do this locally

**Review by iethree — APPROVED** [review]
LGTM.

**Review by nemanjaglumac — APPROVED** [review]
I've removed the required checks and set the `backport` label.
@uladzimirdev all should be good now.

🚢 it

**Pull request merged**